app/controllers/employees_controller.rb

```diff
@@ -5,4 +5,33 @@ class EmployeesController < ApplicationController
   def index
     @employees = Employee.all
   end
+
+  def new
+    @employees = Employee.new
+  end
+
+  def show
+
+  end
+
+  def create
+    data = params.require(:employee)
+    employee = Employee.new
+    employee.nameEmployee = data["nameEmployee"]
+    employee.emailEmployee = data["emailEmployee"]
+    employee.positionEmployee = data["positionEmployee"]
+    employee.attendanceDate = data["attendanceDate"]
+    employee.administrator_id = 1
```

Comment:
you can define a private method called employee_params and invoke it Employee.new(employee_params)
```ruby
private

def employee_params
  params.require(:employee).permit(:nameEmployee, :emailEmployee, :positionEmployee, :attendance_date, :administrator_id)
end
```